User/services/UserService.ts

```diff
@@ -0,0 +1,47 @@
+import  { UserModel }  from '../models/UserModel';
+import { UserDAO } from '../data-access/UserDAO';
+import { User } from '../types/userTypes';
+
+export class UserService {
+
+	public static async getAllUsers(): Promise<UserModel[]> {
+		try {
```

Comment:
Actually you can omit using of try catch blocks in the `userService`, if you really need it. Because errors will be caught in the controller anyway.